hack/patch-crd/main.go

```diff
@@ -0,0 +1,78 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"path/filepath"
+
+	"github.com/Jeffail/gabs"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	crdsDirectory = "../../config/crds"
+	pathToRemove  = "spec.validation.openAPIV3Schema.properties.spec.properties.podTemplate"
+)
+
+func main() {
```

Comment:
Would be nice here to have a comment (or a README) that describes why we need this script.

Reply:
Absolutely.